addhost.php

```diff
@@ -197,7 +205,7 @@
     }//end if
 
     try {
-        $device_id = addHost($host, $snmpver, $port, $transport, $poller_group, $force_add, $port_assoc_mode, $additional);
+        $device_id = addHost($host, $snmpver, $port, $transport, $poller_group, $force_add, $port_assoc_mode, $additional, $within_poller_groups);
```

Comment:
You should pass this as a key in the $additional array.

Reply:
Done